_examples/xkcdsearch/store.go

```diff
@@ -207,7 +207,6 @@ func (s *Store) buildQuery(query string, after ...string) io.Reader {
 
 	b.WriteString("\n}")
 
-	// fmt.Printf("%s\n", b.String())
```

Comment:
I'd rather keep this one, sometimes you just wanna look at the query, and until there's the (planned) support for out-of-the-box logging implemented, you would have to add this, instead of just uncommenting...?

Reply:
Done!